conans/client/cmd/uploader.py

```diff
@@ -244,6 +245,18 @@ def upload_package(self, pref, prev_bundle, remote):
         self._app.remote_manager.upload_package(pref, cache_files, remote)
         duration = time.time() - t1
         self._output.debug(f"Upload {pref} in {duration} time")
+        for cache_file in cache_files:
+            self._local_cache_file(pref, cache_file, remote)
+
+    def _local_cache_file(self, pref, cache_file, remote):
+        approved_files = ["conaninfo.txt", "conan_package.tgz", "conanmanifest.txt"]
+        if cache_file not in approved_files:
+            return
+        _download_cache = self._app.global_conf.get("core.download:download_cache")
+        if _download_cache and os.path.isabs(_download_cache):
+            url = self._app.remote_manager._auth_manager._get_rest_client(remote)._get_api().router.package_file(pref, cache_file)
+            cached_path = DownloadCache(_download_cache).cached_path(url)
```

Comment:
Is someone really doing a copy to the cache? I am missing it. This would be the critical part, my biggest concern about this is the locking, the download cache is concurrent-safe

Reply:
No, only printint for now, ran out of time (Got too sleepy) before being able to add that - will iterate further next week in between other tasks